recipes/django-large-image/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "django-large-image" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/django-large-image-{{ version }}.tar.gz
+  sha256: 3c1f6b576d98bcfc0d2911f5fc625cc3acc2cd66909f0c79ad7009b3173cd61a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - django
+    - djangorestframework
+    - drf-yasg
+    - filelock
+    - large-image >=1.14
+    - python >=3.8
+
+test:
+  imports:
+    - django_large_image
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/girder/django-large-image
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: 'Dynamic tile server in Django built on top of large-image (and GDAL)'
+  description: |
+    django-large-image is an abstraction of large-image for use with
+    django-rest-framework providing viewset mixins for endpoints to work with
+    large images (Cloud Optimized GeoTiffs or medical image formats) in Django.
+
+extra:
+  recipe-maintainers:
+    - banesullivan
+    - manthey
+    - kwrobot
```

Comment:
@kwrobot, are you happy to be listed as a maintianer here?

Reply:
@kwrobot, probably won't respond - @kwrobot is a robot account from my employer, @Kitware, that we try to put on our packages to avoid 🚌 factors

Do you need me to get the @kwrobot account to respond?

Reply:
Ah right, that's fine. It's mostly to prevent unknowing users being added but I'm happy as-is. 👍